packages/status-react/src/routes/chat/components/chat-message/index.tsx

```diff
@@ -77,14 +77,14 @@ export const ChatMessage = (props: Props) => {
   const userProfileDialog = useDialog(UserProfileDialog)
 
   const handleMessageSubmit = (message: string) => {
-    client.community.get(chatId).sendTextMessage(
+    client.community.chats.get(chatId).sendTextMessage(
```

Comment:
Non-blocking: I've noticed the new `getChat` method, but not sure atm if we'll use that or use `chat` from `_chats` getter directly.